project/DistributionPackage.scala

```diff
@@ -552,11 +576,13 @@ object DistributionPackage {
           graalDir / "bin" / "gu.cmd"
       }
       val javaHomeFile = executableFile.getParentFile.getParentFile
-      val javaHome = javaHomeFile.toPath.toAbsolutePath
+      val javaHome     = javaHomeFile.toPath.toAbsolutePath
       val command =
         executableFile.toPath.toAbsolutePath.toString +: arguments
 
-      log.debug(s"Running $command in $graalDir with JAVA_HOME=${javaHome.toString}")
+      log.debug(
```

Comment:
I'm pretty sure it is the result of running `scalafmtAll`. We may not check the formatting of SBT files during the CI, and this file was not properly formatted since the beginning

Reply:
@4e6 is correct, yes. Ideally we would add that check at some point.

Reply:
But I am running `scalafmtAll` in the `/` project from time to time - and it never formats any `sbt` files for me! Or am I hallucinating?